packages/grid/x-grid-modules/src/models/gridOptions.tsx

```diff
@@ -245,7 +250,7 @@ export interface GridOptions {
 export const DEFAULT_GRID_OPTIONS: GridOptions = {
   rowHeight: 52,
   headerHeight: 56,
-  scrollbarSize: 15,
+  scrollbarSize: 10,
```

Comment:
Do we have a logic to compute the size of the scrollbar? It changes with the zoom level.

Reply:
Default is 15
```suggestion
  scrollbarSize: 15,
```

Reply:
We can use https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/Tabs/ScrollbarSize.js or https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/utils/getScrollbarSize.ts to get the size, depending on if we want to handle dynamic changes.